docs/quickstart/README.md

```diff
@@ -107,143 +107,12 @@ Additionally, we have sample server-statefulset files in the directory `server-s
 
 These steps will be different depending on what deployment scheme makes sense for you. Note we have deprecated support of the use case where parts of Tornjak run on the same container as SPIRE. 
 
-Currently, we support two deployment schemes: 
+Currently, we support the following deployment scheme: 
 
-1. Only the Tornjak backend (to make Tornjak API calls)  is run as a separate container on the same pod that exposes only one port (to communicate with the Tornjak backend). It requires more deployment steps to deploy or use the frontend. However, this deployment type is fully-supported, has a smaller sidecar image without the frontend components, and ensures that the frontend and backend share no memory. 
-2. The Tornjak frontend (UI) and backend run in the same container that exposes two separate ports (one frontend and one backend). This is useful for getting started with Tornjak with minimal deployment steps. 
+1. Only the Tornjak backend (to make Tornjak API calls)  is run as a separate container on the same pod that exposes only one port (to communicate with the Tornjak backend). This deployment type is fully-supported, has a smaller sidecar image without the frontend components, and ensures that the frontend and backend share no memory. 
 
 Choose one of the below to easily copy in the right server-statefulset file for you. 
```

Comment:
Change with "Using the option below, easily copy the statefulset in the right server-statefulset file."